### DISCUSSION TITLE
how to add orther callback in login' response

### DISCUSSION THREAD

**Discussion by daniel360-luck** [fastapi-users/fastapi-users#1156] [Q&A]
Thanks for you provide this useful tool.
I have a question: 
login module in my axios(frontend) need more once callback(not only "access_token", "token_type"), like{"msg":"ok" or "success"}, but how can I add these in?
🙏

**Comment by frankie567** [comment]
If you need to return a specific response after login, you'll need to implement your own transport. Here is the default implementation for the JWT transport: https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/authentication/transport/bearer.py

You can override the `get_login_response` method to customize it:

https://github.com/fastapi-users/fastapi-users/blob/024e6cd4516a55c2205fc58f719db1e581dd8078/fastapi_users/authentication/transport/bearer.py#L25-L26
